libnetwork/iptables/firewalld.go

```diff
@@ -56,7 +57,11 @@ func firewalldInit() error {
 	if !firewalldRunning {
 		connection.sysconn.Close()
 		connection = nil
+	} else {
+
+		log.G(context.TODO()).Debugf("Firewalld is currently in state running")
 	}
```

Comment:
Nit: please try to match the code style of the file/function you are editing.
```suggestion
	} else {
		log.G(context.TODO()).Debugf("Firewalld is currently in state running")
	}
```